package.json

```diff
@@ -20,19 +20,21 @@
   "homepage": "https://github.com/theofficetroll/mark-proxy#readme",
```

Comment:
May be worth switching this to be the readme within the organization repo rather than your personal repo?

Reply:
I'm going to point it at the organization repo for my proxy for now instead of my personal fork.